pkgs/desktops/gnome-3/3.24/misc/gnome-tweak-tool/default.nix

```diff
@@ -15,14 +13,17 @@ in stdenv.mkDerivation rec {
   makeFlags = [ "DESTDIR=/" ];
 
   buildInputs = [ pkgconfig gtk3 glib intltool itstool libxml2
-                  gnome3.gsettings_desktop_schemas makeWrapper file
+                  gnome3.gsettings_desktop_schemas file
                   gdk_pixbuf gnome3.defaultIconTheme librsvg
-                  libnotify gnome3.gnome_shell
+                  libnotify gnome3.gnome_shell python2Packages.pygobject3
                   libsoup gnome3.gnome_settings_daemon gnome3.nautilus
-                  gnome3.gnome_desktop wrapGAppsHook
-                  python2Packages.pygobject3.dev pythonEnv gobjectIntrospection ];
+                  gnome3.gnome_desktop wrapGAppsHook gobjectIntrospection
+                ];
 
-  PYTHONPATH = "$out/${pythonEnv.python.sitePackages}";
+  preFixup = ''
+    gappsWrapperArgs+=(
+      --prefix PYTHONPATH : "$out/${python2Packages.python.sitePackages}")
```

Comment:
This should not be necessary if we add python as a dependency according to [docs](https://github.com/NixOS/nixpkgs/blob/cb2f2aa563a5be0eca70d3a16a9351a7d419438f/doc/languages-frameworks/python.md#interpreters). It should probably be listed anyway, otherwise the pygobject dependency could not be used.